### ISSUE TITLE
TabFolder offset value is not '0' when the tab transition is finished

### ISSUE THREAD

**Issue by ahmadov** [eclipsesource/tabris-js#1952]
### Problem description

The offset value of TabFolder is not '0' when the tab transition finished.

### Expected behavior

After the tab transition stops the offset value should be '0'.
On iOS the offset value is '0' as expected.

### Environment

- Tabris.js version: 2.x, 3.x
- Device: any
- OS: Android

### Code snippet

```jsx
import {Tab, TabFolder, TextView, contentView, Composite, Properties } from 'tabris';
import { component, property } from 'tabris-decorators';

@component
class CustomComponent extends Composite {
  @property public currentOffset: string = '';

  constructor(properties?: Properties<Composite>) {
    super();
    this.set(properties).append(
      <TabFolder id="tabFolder" stretch paging tabBarLocation='hidden'>
        <Tab>
          <TextView center font='20px'>Page 1/3</TextView>
          <TextView centerX top={'prev() 10'} font='20px' bind-text='currentOffset'></TextView>
        </Tab>
        <Tab>
          <TextView center font='20px'>Page 2/3</TextView>
          <TextView centerX top={'prev() 10'} font='20px' bind-text='currentOffset'></TextView>
        </Tab>
        <Tab>
          <TextView center font='20px'>Page 3/3</TextView>
          <TextView centerX top={'prev() 10'} font='20px' bind-text='currentOffset'></TextView>
        </Tab>
      </TabFolder>
    );

    const tabFolder = this._find('#tabFolder').only();
    tabFolder.on('scroll', ({ offset }: any) =>
      this.currentOffset = 'offset:\n' + offset.toFixed(2));
  }
}

contentView.append(
  <CustomComponent stretch/>
);
```

**Comment by tbuschto** [comment]
Indeed it sometimes stick to values >= 0.5, meaning even Math.round doesn't help.

**Comment by mpost** [comment]
Fixed in the upcoming 3.3 release.
